src/DFAMin.hs

```diff
@@ -1,218 +1,258 @@
-{-# OPTIONS_GHC -fno-warn-name-shadowing #-}
-
 {-# LANGUAGE CPP #-}
 
 module DFAMin (minimizeDFA) where
 
-import AbsSyn
+import           AbsSyn
```

Comment:
I'm not a maintainer here, so feel free to ignore, but my personal preference would be to reduce amount of whitespace changes, even when existing indentation / alignment happens to be inconsistent. `git blame` is too valuable to handicap, especially for such old projects.

Reply:
Done. I'd be in favour of some standardization, which this project lacks, but that's definitely beyond the scope of this PR. ^^